.github/workflows/build.yml

```
@@ -30,31 +30,53 @@ jobs:
             echo "HOST_TRIPLE=aarch64-unknown-linux-gnu" >> $GITHUB_ENV
         fi
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Build the Docker image build environment
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:35: Double quote to prevent globbing and word splitting

(shellcheck)